pkg/airgapped/plugin_bundle_upload.go

```diff
@@ -82,7 +81,7 @@ func (o *UploadPluginBundleOptions) UploadPluginBundle() error {
 	}
 
 	log.Infof("---------------------------")
-	log.Infof("successfully published all plugin images to %q", fmt.Sprintf("%s%s", o.DestinationRepo, manifest.RelativeInventoryImagePathWithTag))
+	log.Infof("successfully published all plugin images to %q", filepath.Join(o.DestinationRepo, manifest.RelativeInventoryImagePathWithTag))
```

Comment:
I have a feeling that this might cause issues on the Windows machine as `filepath.Join` uses `\` for windows machines. Can we verify this once on windows?

Reply:
@anujc25  I have looked into filepath.Join() functionality- Its implemention is OS aware.

Here's how it works for Mac, Linux, and Windows:

Mac and Linux: For these Unix-based operating systems, filepath.Join() uses / as the path separator, which is the standard for Unix-like systems. If you're running this function on either Mac or Linux, it will automatically use / to join the paths. For example, filepath.Join("/home", "user", "documents") would return "/home/user/documents".

Windows: Windows uses a different default path separator, the backslash \. When filepath.Join() is used in Windows, it will use \ to join the paths. For example, filepath.Join("C:\\", "Users", "Username", "Documents") would return "C:\\Users\\Username\\Documents".

Reply:
Considering this is a container image path and not the file path. if we use `filepath.Join` on windows it will use `\` in the container image which will make the container image invalid as the container image path should always have `/`. Correct?